content/work/coachhub.rst

```diff
@@ -50,24 +50,28 @@ planning:
       before we began development, and throughout the project —
       remaining up-to-date and in control of the project cost and scope.'
   - title: 'Data Models'
-    icon: 'miriam'
+    icon: 'site-map'
     text: 'Placeholder text...'
```

Comment:
oops, I removed it in the short term if they can't get to it today. Would be easy to add back in.